promscale/installation/index.md

```diff
@@ -30,36 +25,37 @@ and adds its public schemas to the search path. This makes querying Promscale
 data easier. The public schemas that Promscale adds are: `ps_tag`, `prom_api`,
 `prom_metric`, `ps_trace`. 
 </highlight>
+ 
 
 ## Install Promscale without instrumentation
-If you have Prometheus or OpenTelemetry installed, you can install Promscale
-on these environments:
-
-*   **Kubernetes**:  use [Helm charts][promscale-install-helm] or [a manifest][promscale-install-k8s-manifest] file.
-*   **Docker**: use a [pre-built Docker container][promscale-install-docker].
-*   **VM or bare metal**: use the [source][promscale-install-source] file.
+If you want to install Promscale on your infrastructure where you have already installed Prometheus or OpenTelemetry, you can do that on:
 
-You can also use our [prom-migrator tool][promscale-install-prom-migrator] to
-migrate your existing Prometheus data into Promscale.
+* Kubernetes using [helm charts][helm-charts], or a [manifest file][manifest-file]
+* [Pre-built containers][docker]
```

Comment:
Pre-built containers phrase is confusing. As container comes into life only on `docker run`, so for simplicity let's say `Docker` as before. For reference, Grafana and VM use docker as the title/keyword to represent container-related installation.